qa/rpc-tests/getblocktemplate_proposals.py

```diff
@@ -140,11 +140,18 @@ def run_test(self):
         mark_logs(("active chain height = %d: testing before sidechain fork" %  self.nodes[0].getblockcount()), self.nodes, DEBUG_MODE)
         self.doTest(sc_fork_reached)
 
+        # reach the height where the next block is the last before the fork point where certificates are supported
+        self.nodes[0].generate((MINIMAL_SC_HEIGHT - 1) - 200 - 2) 
```

Comment:
I would suggest to use `currentHeight` retrieved from RPC instead of hardcoding `- 1) - 200 - 2`

Reply:
Done